examples/deployment/scripts/transfer/gold_sc.js

```diff
@@ -34,18 +34,18 @@ async function run(runtimeEnv, deployer) {
     //console.error(e);
   }
 
-  // Transaction for ALGO - Contract : '3-gold-asc.teal'
-  const logicSignature = await deployer.getLogicSignature("3-gold-asc.teal", []);
+  // Transaction for ALGO - Contract : '3-gold-delegated-asc.teal'  (Delegated Approval Mode)
+  const logicSignature = await deployer.getLogicSignature("3-gold-delegated-asc.teal", []);
   logicSignature.sign(goldOwnerAccount.sk);
   // Will pass - As according to .teal logic, amount should be <= 100
   // Transaction PASS
-  const tranDetails =  await transferMicroAlgosLsig(deployer, goldOwnerAccount, bobAccount.addr, 58, logicSignature);
+  const tranDetails =  await transferMicroAlgosLsig(deployer, goldOwnerAccount.addr, bobAccount.addr, 58, logicSignature);
   console.log(tranDetails);
 
   // Gets rejected by logic - As according to .teal logic, amount should be <= 100
   // Transaction FAIL
   try {
-    await transferMicroAlgosLsig(deployer, goldOwnerAccount, bobAccount.addr, 580, console.logicSignature);
+    await transferMicroAlgosLsig(deployer, goldOwnerAccount.addr, bobAccount.addr, 580, logicSignature);
   } catch(e) {
     console.log('Transaction Failed - rejected by logic')
```

Comment:
same here. Let's use `console.error` and group common things in a function. (maybe we can reuse that function from other in other scripts as well by putting it in `scripts/transfer/common.js`.

Reply:
Done